db/migrate/20231031164710_add_slug_to_organisms.rb

```diff
@@ -0,0 +1,5 @@
+class AddSlugToOrganisms < ActiveRecord::Migration[6.1]
+  def change
+    add_column :organisms, :slug, :string
```

Comment:
maybe add  a description via comment: 'a human readable unique identifier of the organism"
also maybe missing a unique constraint?

Reply:
Agreed, adding both and an index on this because I use it in queries a lot.